openstack/sap-seeds/templates/flavor-seed.yaml

```diff
@@ -13,477 +13,385 @@ spec:
     disk: 1
     is_public: true
     extra_specs:
-      "vmware:hv_enabled": "True"
-      "hw_video:ram_max_mb": "4"
```

Comment:
idk why, but this flavor has only 4 MB of video RAM and would now get 16 MB. With 16 MB it would be larger than 512 MB and we usually set `ram` to "$fullsize - $videoram"

Reply:
Fixed it, now one can pass the macro a dictionary with values for a flavor which override the defaults.